src/cloud.js

```diff
@@ -749,3 +749,22 @@ Cloud.prototype.updateNotes = function (
     );
 };
 
+Cloud.prototype.updateProjectName = function (
+	projectName,
+    newName,
+    onSuccess,
+    onError
+) {
+    this.withCredentialsRequest(
+        'POST',
+        '/projects/%username/' +
+            encodeURIComponent(projectName) +
+            '/metadata',
+        onSuccess,
+        onError,
+        'Could not update project notes',
```

Comment:
Shouldn't this say, "Could not update project name"?

Reply:
It should indeed. Thanks for noticing!